src/main/java/com/okta/tools/helpers/FileHelper.java

```diff
@@ -0,0 +1,89 @@
+package com.okta.tools.helpers;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public final class FileHelper {
+    private static final String USER_HOME = System.getProperty("user.home");
+
+    /**
+     * Gets a directory with the given path, creating it if it doesn't exist
+     *
+     * @param path The path of the directory to be returned
+     * @return The path of the directory
+     * @author Andrei Hava
+     * @since 02/14/2018
+     */
+    private static Path getDirectory(String path) {
+        File dir = new File(path);
+        dir.mkdir();
+
+        return dir.toPath();
+    }
+
+    /**
+     * Gets the path of the AWS directory (USER_HOME/.aws)
+     *
+     * @return The path of the AWS directory
+     * @author Andrei Hava
+     * @since 02/14/2018
+     */
+    public static Path getAwsDirectory() {
+        return getDirectory(USER_HOME + "/.aws/");
```

Comment:
:bug: It's unsafe to build paths with String concatenation.

Recommendation: move the pre-existing Path-based methods from OktaAwsCliAssumeRole.